internal/slicer/slicer.go

```diff
@@ -242,17 +254,28 @@ func Run(options *RunOptions) (*Report, error) {
 			if err != nil {
 				return nil, err
 			}
-			err = report.Add(slice, entry)
-			if err != nil {
-				return nil, err
+
+			// Do not add paths with "until: mutate".
+			if pathInfo.Until != setup.UntilMutate {
+				err = report.Add(slice, entry)
+				if err != nil {
+					return nil, err
+				}
 			}
 		}
 	}
 
 	// Run mutation scripts. Order is fundamental here as
 	// dependencies must run before dependents.
 	checkWrite := func(path string) error {
-		if !pathInfos[path].Mutable {
+		pi, ok := pathInfos[path]
```

Comment:
Isn't this called pathInfo everywhere above?

Reply:
This function does not exist anymore after the changes.